### ISSUE TITLE
About use

### ISSUE THREAD

**Issue by misszhou966** [ddsediri/IterativePFN#8]
Hello, I'm sorry to bother you again, I want to use your network structure to train my data to achieve the task I want to achieve, but the log I get with the same parameters is like this, the effect is very unsatisfactory, do you have any suggestions? Thank you very much for your help.
INFO: Start training...
INFO: Current epoch validation loss: 1.355746
INFO: Current epoch validation loss: 1.330306
INFO: Current epoch validation loss: 1.355451
INFO: Current epoch training loss: 3.213099
INFO: Current epoch training loss: 3.208412
INFO: Current epoch training loss: 3.228679
INFO: Current epoch training loss: 2.886272
INFO: Current epoch training loss: 2.898693
INFO: Current epoch training loss: 2.888039
INFO: Current epoch training loss: 2.801729
INFO: Current epoch training loss: 2.807481
INFO: Current epoch training loss: 2.806274
INFO: Current epoch training loss: 2.748672
INFO: Current epoch training loss: 2.731634
INFO: Current epoch training loss: 2.744804
INFO: Current epoch validation loss: 1.260837
INFO: Current epoch validation loss: 1.332064
INFO: Current epoch validation loss: 1.276065
INFO: Current epoch training loss: 2.717337
INFO: Current epoch training loss: 2.716443
INFO: Current epoch training loss: 2.719618
INFO: Current epoch training loss: 2.694257
INFO: Current epoch training loss: 2.699254
INFO: Current epoch training loss: 2.711572
INFO: Current epoch training loss: 2.660917
INFO: Current epoch training loss: 2.661672
INFO: Current epoch training loss: 2.660765
INFO: Current epoch training loss: 2.648638
INFO: Current epoch training loss: 2.654757
INFO: Current epoch training loss: 2.653681
INFO: Current epoch training loss: 2.641084
INFO: Current epoch training loss: 2.654339
INFO: Current epoch training loss: 2.655447
INFO: Current epoch validation loss: 1.236746
INFO: Current epoch validation loss: 1.327998
INFO: Current epoch validation loss: 1.262958
INFO: Current epoch training loss: 2.635180
INFO: Current epoch training loss: 2.636929
INFO: Current epoch training loss: 2.649486
INFO: Current epoch training loss: 2.644213
INFO: Current epoch training loss: 2.645515
INFO: Current epoch training loss: 2.622950
INFO: Current epoch training loss: 2.619204
INFO: Current epoch training loss: 2.623927
INFO: Current epoch training loss: 2.627945
INFO: Current epoch training loss: 2.626814
INFO: Current epoch training loss: 2.622999
INFO: Current epoch training loss: 2.636228
INFO: Current epoch training loss: 2.610208
INFO: Current epoch training loss: 2.620833
INFO: Current epoch training loss: 2.609075
INFO: Current epoch validation loss: 1.223196
INFO: Current epoch validation loss: 1.324197
INFO: Current epoch validation loss: 1.254352
INFO: Current epoch training loss: 2.603472
INFO: Current epoch training loss: 2.606685
INFO: Current epoch training loss: 2.616014
INFO: Current epoch training loss: 2.599103
INFO: Current epoch training loss: 2.620140
INFO: Current epoch training loss: 2.609823
INFO: Current epoch training loss: 2.602850
INFO: Current epoch training loss: 2.593271
INFO: Current epoch training loss: 2.612407
INFO: Current epoch training loss: 2.595728
INFO: Current epoch training loss: 2.599696
INFO: Current epoch training loss: 2.600771
INFO: Current epoch training loss: 2.603282
INFO: Current epoch training loss: 2.590809
INFO: Current epoch training loss: 2.589800
INFO: Current epoch validation loss: 1.224958
INFO: Current epoch validation loss: 1.324403
INFO: Current epoch validation loss: 1.257810
INFO: Current epoch training loss: 2.586303
INFO: Current epoch training loss: 2.597836
INFO: Current epoch training loss: 2.598653
INFO: Current epoch training loss: 2.596004
INFO: Current epoch training loss: 2.582781
INFO: Current epoch training loss: 2.585327
INFO: Current epoch training loss: 2.568661
INFO: Current epoch training loss: 2.574789
INFO: Current epoch training loss: 2.577212
INFO: Current epoch training loss: 2.583612
INFO: Current epoch training loss: 2.581432
INFO: Current epoch training loss: 2.576155
INFO: Current epoch training loss: 2.582904
INFO: Current epoch training loss: 2.565365
INFO: Current epoch training loss: 2.576646
INFO: Current epoch validation loss: 1.234995
INFO: Current epoch validation loss: 1.323563
INFO: Current epoch validation loss: 1.263268
INFO: Current epoch training loss: 2.574958
INFO: Current epoch training loss: 2.567900
INFO: Current epoch training loss: 2.575992
INFO: Current epoch training loss: 2.574017
INFO: Current epoch training loss: 2.579748
INFO: Current epoch training loss: 2.568510
INFO: Current epoch training loss: 2.572126
INFO: Current epoch training loss: 2.564318
INFO: Current epoch training loss: 2.565993
INFO: Current epoch training loss: 2.559158
INFO: Current epoch training loss: 2.561340
INFO: Current epoch training loss: 2.560112
INFO: Current epoch training loss: 2.566038
INFO: Current epoch training loss: 2.574062
INFO: Current epoch training loss: 2.560465
INFO: Current epoch validation loss: 1.223749
INFO: Current epoch validation loss: 1.325153
INFO: Current epoch validation loss: 1.253113
INFO: Current epoch training loss: 2.565150
INFO: Current epoch training loss: 2.569508
INFO: Current epoch training loss: 2.551525
INFO: Current epoch training loss: 2.558621
INFO: Current epoch training loss: 2.568434
INFO: Current epoch training loss: 2.559792
INFO: Current epoch training loss: 2.560946
INFO: Current epoch training loss: 2.561964
INFO: Current epoch training loss: 2.576425
INFO: Current epoch training loss: 2.561572
INFO: Current epoch training loss: 2.553172
INFO: Current epoch training loss: 2.562437
INFO: Current epoch training loss: 2.550814
INFO: Current epoch training loss: 2.560851
INFO: Current epoch training loss: 2.548186
INFO: Current epoch validation loss: 1.220110
INFO: Current epoch validation loss: 1.324611
INFO: Current epoch validation loss: 1.253090
INFO: Current epoch training loss: 2.554108
INFO: Current epoch training loss: 2.555075
INFO: Current epoch training loss: 2.556961
INFO: Current epoch training loss: 2.561786
INFO: Current epoch training loss: 2.560792
INFO: Current epoch training loss: 2.553971
INFO: Current epoch training loss: 2.567475
INFO: Current epoch training loss: 2.558358
INFO: Current epoch training loss: 2.553187
INFO: Current epoch training loss: 2.565339
INFO: Current epoch training loss: 2.561377
INFO: Current epoch training loss: 2.550573
INFO: Current epoch training loss: 2.547145
INFO: Current epoch training loss: 2.556738
INFO: Current epoch training loss: 2.541249
INFO: Current epoch validation loss: 1.228358
INFO: Current epoch validation loss: 1.325231
INFO: Current epoch validation loss: 1.258611
INFO: Current epoch training loss: 2.548273
INFO: Current epoch training loss: 2.553710
INFO: Current epoch training loss: 2.550095
INFO: Current epoch training loss: 2.545655
INFO: Current epoch training loss: 2.547244
INFO: Current epoch training loss: 2.549295
INFO: Current epoch training loss: 2.547032
INFO: Current epoch training loss: 2.553643
INFO: Current epoch training loss: 2.547189
INFO: Current epoch training loss: 2.546811
INFO: Current epoch training loss: 2.559923
INFO: Current epoch training loss: 2.545727
INFO: Current epoch training loss: 2.550506
INFO: Current epoch training loss: 2.546952
INFO: Current epoch training loss: 2.535547
INFO: Current epoch validation loss: 1.214735
INFO: Current epoch validation loss: 1.325262
INFO: Current epoch validation loss: 1.245251
INFO: Current epoch training loss: 2.547129
INFO: Current epoch training loss: 2.537701
INFO: Current epoch training loss: 2.548097
